### PR TITLE
Children for Locally Factorable

### PR DESCRIPTION
**Pull request by enadeau**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -326,13 +326,13 @@ done here for sake of brevity in this readme!
...
+       3 -> (0,)
        tiling is locally factorable
-       +-+-+-+
-       | |●| |
-       +-+-+-+
-       | | |1|
+       +-+-+-+            +-+
+       | |●| |         ↝  |1|
+       +-+-+-+            +-+
+       | | |1|            1: Av(120)
```
Comment (christianbean) [inline README.rst:335]: The paragraph after this needs updated with the latest change.

File: tilings/strategy_pack.py [hunk 2]
```diff
@@ -23,17 +24,21 @@ class TileScopePack(StrategyPack):
     # Method to add power to a pack
     # Pack are immutable, these methods return a new pack.
 
-    def fix_one_by_one(self, basis: Iterable[Perm]) -> "TileScopePack":
+    def inject_basis(self, basis: Iterable[Perm]) -> "TileScopePack":
```
Comment (christianbean) [inline tilings/strategy_pack.py:27]: I would prefer 'add_basis'

Comment (enadeau) [reply tilings/strategy_pack.py]: done

File: README.rst (hunk 1 above)
Comment (enadeau) [reply README.rst:335]: done

**Review by christianbean — APPROVED** [review]

**Pull request merged**